README.md

```diff
@@ -1,7 +1,14 @@
-# Headlamp <img align="right" width=384 src="docs/headlamp_light.svg"> [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7551/badge)](https://www.bestpractices.dev/projects/7551) [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/headlamp-k8s/headlamp/badge)](https://scorecard.dev/viewer/?uri=github.com/headlamp-k8s/headlamp)
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="frontend/src/resources/logo-light.svg">
+    <img src="frontend/src/resources/logo-dark.svg" alt="Headlamp Logo">
```

Comment:
To address the concern (I believe it's typically semantically incorrect to wrap an image in h1), maybe we can include the "Headlamp" text in here for SEO/a11y and then use CSS to hide it visually

Reply:
shouldn't be an issue